tests/Integration/Behaviour/Features/Scenario/Product/Combination/combination_stock_movement_history.feature

```diff
@@ -0,0 +1,136 @@
+@restore-all-tables-before-feature
+@clear-cache-before-feature
+@legacy-product-type
+@product-combination
+@combination-stock-movement-history
+Feature: Search stock movement history from Back Office (BO)
+  As a BO user
+  I need to be able to search stock movement history for a combination from BO
+
+  Background:
+    Given email sending is disabled
+    And the current currency is "USD"
+    And country "US" is enabled
+    And the module "dummy_payment" is installed
+    And I am logged in as "test@prestashop.com" employee
+    And there is customer "testCustomer" with email "pub@prestashop.com"
+    And customer "testCustomer" has address in "US" country
+    And a carrier "default_carrier" with name "My carrier" exists
+#    Given there is a product in the catalog named "product1" with a price of 17.0 and 1000 items in stock
+#    And product "product1" has a combination named "combination1" with 100 items in stock
+    Given I add product "product1" with following information:
+      | name[en-US] | universal T-shirt |
+      | type        | combinations      |
+    And language with iso code "en" is the default one
+    And attribute group "Size" named "Size" in en language exists
+    And attribute group "Color" named "Color" in en language exists
+    And attribute "S" named "S" in en language exists
+    And attribute "M" named "M" in en language exists
+    And attribute "L" named "L" in en language exists
+    And attribute "White" named "White" in en language exists
+    And attribute "Black" named "Black" in en language exists
+    And attribute "Blue" named "Blue" in en language exists
+    And attribute "Red" named "Red" in en language exists
+    And I generate combinations for product "product1" using following attributes:
+      | Size  | [S,M]              |
+      | Color | [White,Black,Blue] |
+    And product "product1" should have following combinations:
+      | id reference   | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SWhite | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 0        | true       |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | false      |
+      | product1Blue   | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhite | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlack | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    Given I update combination "product1SBlack" stock with following details:
```

Comment:
Stock movements are already tested in other scenarios :
- tests/Integration/Behaviour/Features/Scenario/Product/update_stock.feature
- tests/Integration/Behaviour/Features/Scenario/Product/Combination/combination_update_stock.feature
- tests/Integration/Behaviour/Features/Scenario/Order/order_stock.feature => https://github.com/PrestaShop/PrestaShop/pull/27626

There is no edition in stock movement history, because it's only a view extracted from stock movements.
That's why here we need to create a few stock movements, from orders and employee editions, in order to test the history behavior properly.
History is paginable, so it's actually faster to stack up cases and navigate through.